JavaScript/Post_Tweets/index.js

```diff
@@ -0,0 +1,38 @@
+var Twit = require('twit')
+const fs = require('fs');
+var T = new Twit({
+  consumer_key:         'YOUR_CONSUMER_KEY',
+  consumer_secret:      'YOUR_CONSUMER_SECRET',
+  access_token:         'YOUR_ACCESS_TOKEN',
+  access_token_secret:  'YOUR_ACCESS_SECRET',
+})
```

Comment:
Use [dotenv NPM Package](https://www.npmjs.com/package/dotenv) to handle sensitive stuff like these.